react-native-ssi/example/src/App.tsx

```diff
@@ -47,6 +53,30 @@ export function App() {
             >
               <Text style={styles.buttonText}>Expand DIDDoc</Text>
             </TouchableOpacity>
+            <TouchableOpacity
+              disabled={didKey == ''}
+              style={styles.button}
+              onPress={() => {
+                if (typeof privateJwk == 'undefined') {
+                  return;
+                }
+
+                createVerifiableCredential()
+                  .then((vc) => {
+                    return signVerifiableCredentialJWT(didKey, privateJwk, vc);
+                  })
+                  .then((jwt) => {
+                    return verifyVerifiableCredentialJWT(didKey, didKey, jwt);
```

Comment:
is this a common scenario that the didKey is the same thing as they keyid? should this be one param?

Reply:
I don't really know. Perhaps @decentralgabe or @andresuribe87 would know?